src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -129,26 +158,20 @@ The data you can expect back for both the initial grant code exchange and subseq
 }
 ```
 
-#### How to use for requests
+### Auth Tokens
 
-When making requests to the Sentry API, you use the token just like you would when you're typically making [API requests](/api/auth/). Tokens are associated with the installation, meaning they have access to the Sentry organization that installed your integration.
+Auth Tokens are what the Integration Platform calls tokens. Auth Tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
```

Comment:
I'm confused by this bit. Lemme know if this edit doesn't work and we might need to talk it through.

```suggestion
Auth Tokens are referred to as _tokens_ in the Integration Platform. Auth tokens are passed using an auth header and are used to authenticate as a user account with the API. In the case of the Integration Platform, these tokens are used to authenticate as an application (with access to a specific organization) instead of a user.
```